diag_test.py

```
@@ -398,31 +449,128 @@ def custom_sort(label):
+                    
+                    {"Metric": "Risk Ratio (RR)", "Value": f"{rr:.4f}", 
+                     "95% CI": rr_ci_display, 
+                     "Interpretation": f"Risk in {label_exp} is {rr:.2f}x that of {label_unexp}"},
+                    
+                    {"Metric": "Odds Ratio (OR)", "Value": f"{or_value:.4f}", 
+                     "95% CI": or_ci_display, 
+                     "Interpretation": f"{or_badge} Odds of event in {label_exp} vs {label_unexp}"},
+                    
+                    {"Metric": "Absolute Risk Reduction (ARR)", "Value": f"{arr:.2f}%", 
+                     "95% CI": "-", 
+                     "Interpretation": f"Absolute difference in event rates"},
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)